.github/workflows/ci.yml

```diff
@@ -15,21 +15,21 @@ jobs:
       fail-fast: false
       matrix:
         triplet:
-          - otp: '25'
+          - otp: '26'
             elixir: '1.16'
             rebar3: '3.22.1'
-          - otp: '26'
+          - otp: '27'
             elixir: '1.17'
             rebar3: '3.22.1'
-          - otp: '27'
-            elixir: '1.18'
-            rebar3: '3.23'
+          - otp: '28'
+            elixir: '1.18.4-otp-27'
+            rebar3: '3.25'
           - otp: 'master'
             elixir: 'main'
             rebar3: 'nightly'
     steps:
       - uses: actions/checkout@v2
-      - uses: erlef/setup-beam@v1
+      - uses: erlef/setup-beam@8aa8a857c6be0daae6e97272bb299d5b942675a4
```

Comment:
You could maybe add `# v...` so we humans can relate better (?) 😄

Reply:
I've started using sha per [security incidents](https://www.cisa.gov/news-events/alerts/2025/03/18/supply-chain-compromise-third-party-tj-actionschanged-files-cve-2025-30066-and-reviewdogaction) 

Maybe you're suggesting a comment such that the reader knows which version we're pointing to without having to look it up?

Reply:
I'm suggesting a comment, correct. 👍 This is e.g. the way Renovate bot does it. If you want I can help set it up (OTP is using it, and so am I in other projects, for example).